bin/contact.html

```diff
@@ -0,0 +1 @@
+undefined
```

Comment:
this file isn't needed. the bin folder should only contain javascript files. I think you may have run a build in the bin directory and then committed all the files it generated.